src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        ArrayList<Task> tasks = new ArrayList<>();
+        Scanner sc = new Scanner(System.in);
+        printHorizontalLine();
+        System.out.println("  Hello! I'm Duke");
+        System.out.println("  What can I do for you?");
+        printHorizontalLine();
```

Comment:
I like how you have abstracted out the action of printing a horizontal line into a method. However, perhaps you can extend this level of abstraction to the other printing actions as well i.e. maybe you can define the strings as constants and then use a specially defined method to print out those constants, so as to also lessen the use of magic numbers / strings.